plugin.program.autocompletion/addon.xml

```diff
@@ -1,18 +1,21 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.program.autocompletion" name="AutoCompletion for virtual keyboard" version="2.0.1" provider-name="Philipp Temminghoff (phil65),sualfred">
+<addon id="plugin.program.autocompletion" name="AutoCompletion for virtual keyboard" version="2.0.5" provider-name="Philipp Temminghoff (phil65), sualfred, xulek, finkleandeinhorn">
     <requires>
         <import addon="xbmc.python" version="3.0.0" />
-        <import addon="script.module.autocompletion" version="2.0.0"/>
+        <import addon="script.module.autocompletion" version="2.0.3"/>
     </requires>
-    <extension point="xbmc.python.script" library="default.py">
+    <extension point="xbmc.python.script"       library="default.py">
         <provides>executable</provides>
     </extension>
     <extension point="xbmc.python.pluginsource" library="plugin.py" />
+
     <extension point="xbmc.addon.metadata">
         <summary lang="en_GB">AutoCompletion for the virtual keyboard (needs skin support)</summary>
         <description lang="en_GB">AutoCompletion for the virtual keyboard (needs skin support)</description>
+        <summary lang="es_ES">Autocompletar para el teclado virtual (necesita soporte de skin)</summary>
+        <description lang="es_ES">Autocompletar para el teclado virtual (necesita soporte de skin)</description>
         <platform>all</platform>
-        <license>GPL-2.0-or-later</license>
+        <license>GNU GENERAL PUBLIC LICENSE Version 2.1, February 1999</license>
```

Comment:
This is wrong, should be kept to the SPDX license identifier just like it was before.

Reply:
As you can see from the commit diff, the original license was `GPL-2.0-or-later`, and the it was changed to `GNU GENERAL PUBLIC LICENSE Version 2.1, February 1999`, which is a compliant license (later than 2.0).

I will change it back anyways, as this is not actually a relevant change to this PR, and the license has indeed not changed.